drivers/scheduler/k8s/specs/fio-fada-realm/px-storage-class.yaml

```diff
@@ -0,0 +1,29 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: fio-realm-pod-sc
+provisioner: pxd.portworx.com
+parameters:
+  backend: "pure_block"
+  max_iops: "1000"
+  max_bandwidth: "1G"
+  fs: "ext4"
+  pure_fa_pod_name: "firstpod"
```

Comment:
this cannot be hard-coded, needs to be placeholder and we will need to create a new pod and pass it